File-Formats/DocIO/Find-item-in-word-document.md

```diff
@@ -0,0 +1,738 @@
+        }
+        //Save a Word document to the MemoryStream.
+        MemoryStream outputStream = new MemoryStream();
+        document.Save(outputStream, FormatType.Docx);
+        //Save the stream as a file in the device and invoke it for viewing. 
+        Xamarin.Forms.DependencyService.Get<ISave>().SaveAndView("Sample.docx", "application/msword", outputStream);
+    }
+}
+//Download the helper files from the following link to save the stream as file and open the file for viewing in Xamarin platform
+//https://help.syncfusion.com/file-formats/docio/create-word-document-in-xamarin#helper-files-for-xamarin
+{% endhighlight %} 
+
+{% endtabs %}
+
+## Find all items by properties
+
+Using FindAllItemsByProperties API, find all the items in the Word document based on multiple property names and their corresponding values.
+
+The following code example illustrates how to find all the items in Word document based on multiple property names and their corresponding values.
+
+{% tabs %}
+
+{% highlight c# tabtitle="C#" %}
+//Load an existing Word document.
+using (WordDocument document = new WordDocument("Input.docx", FormatType.Docx))
+{
+    string[] propertyNames = { "ContentControlProperties.Title", "ContentControlProperties.Tag" };
+    string[] propertyValues = { "CompanyName", "CompanyName" };
+
+    //Find all block content control by Title and Tag. 
+    List<Entity> blockContentControls = document.FindAllItemsByProperties(EntityType.BlockContentControl,propertyNames,propertyValues);
+
+    //Iterates the block content controls and remove the block content controls.
+    for (int i = 0; i < blockContentControls.Count; i++)
+    {
+        BlockContentControl blockContentControl = blockContentControls[i] as BlockContentControl;
+        blockContentControl.OwnerTextBody.ChildEntities.Remove(blockContentControl);
+    }
+
+    propertyNames = new string[] { "ContentControlProperties.Title", "ContentControlProperties.Tag" };
+    propertyValues = new string[] { "Contact", "Contact" };
+
+    //Find all inline content control by Title and Tag. 
+    List<Entity> inlineContentControls = document.FindAllItemsByProperties(EntityType.InlineContentControl,propertyNames,propertyValues);
+
+    //Iterates the inline content controls and remove the inline content controls.
+    for (int i = 0; i < inlineContentControls.Count; i++)
+    {
+        InlineContentControl inlineContentControl = inlineContentControls[i] as InlineContentControl;
+        inlineContentControl.OwnerParagraph.ChildEntities.Remove(inlineContentControl);
+    }
+
+    propertyNames = new string[] { "CharacterFormat.Bold", "CharacterFormat.Italic" };
+    propertyValues = new string[] { true.ToString(), true.ToString() };
+
+    //Find all bold and italic text.
+    List<Entity> textRanges = document.FindAllItemsByProperties(EntityType.TextRange,propertyNames,propertyValues);
+
+    //Iterates the textRanges and Remove the bold and italic
```

Comment:
//Iterates the textRanges and remove the bold and italic

Reply:
Addressed